### ISSUE TITLE
Inability to create a standard roll due to overridden method in the Actor document class

### ISSUE THREAD

**Issue by krbz999** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#410]
### Describe the bug

I was expecting to be able to use an actor's roll data for creating a roll by writing a standard macro such as this:
```js
const rollData = actor.getRollData();
const formula = "(@attribute.agility.value)d6";
const speaker = ChatMessage.implementation.getSpeaker({actor: actor});
return new Roll(formula, rollData).toMessage({speaker});
```
but it seems that `getRollData` is overridden in the actor document class to return something unrelated to rolls.

The workaround is to use `const rollData = actor.system;`.

Code here: <https://github.com/fvtt-fria-ligan/forbidden-lands-foundry-vtt/blob/main/src/actor/actor-document.js#L41>

### Reproduction

Use the above sample macro.

### System Info

```shell
n/a
```

### Validations

- [X] Done a clean install of the system.
- [X] Disabled all modules.
- [X] Checked if the issue is present in a new world instance (create a new world in foundry using the Forbidden Lands system).
- [X] Not a bug report about the content modules. Reports regarding the content modules go [here instead](https://github.com/aMediocreDad/fbl-content-modules/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).
- [X] Not a "How To"-question (You can ask questions here: [Discussions](https://github.com/fvtt-fria-ligan/forbidden-lands-foundry-vtt/discussions)).

**Comment by aMediocreDad** [comment]
Yeah this is an old choice that was made in ignorance. Not so much a bug as incorrect override of the API. Changing it requires a bit of work.

**Comment by Doresain** [comment]
Fixed this by renaming the overridden method. Regular rolls and the macro above both now work as expected.

Will try to get it into the next release.
